src/core/tests/library/ArgumentComposer.py

```diff
@@ -68,7 +69,8 @@ def get_composed_arguments(self):
             "rebootSetting": self.reboot_setting,
             "classificationsToInclude": self.classifications_to_include,
             "patchesToInclude": self.patches_to_include,
-            "patchesToExclude": self.patches_to_exclude
+            "patchesToExclude": self.patches_to_exclude,
+            "maxPatchPublishDate": self.max_patch_publish_date
```

Comment:
Since this is an optional parameter - tests need to account for it being present and not. To ensure both these high-level cases are considered, the value should not be present by default, and should be dynamically added to the object in the tests that will rely on it.

Reply:
Changed. The test with a value present will be added later, with the status changes